code/modules/client/client procs.dm

```diff
@@ -122,6 +122,16 @@ var/next_external_rsc = 0
 
 	. = ..()	//calls mob.Login()
 
+	if (connection == "web")
+		if (!config.allowwebclient)
+			src << "Web client is disabled"
+			del(src)
+			return 0
+		if (config.webclientmembersonly && !is_content_unlocked())
+			src << "Sorry, but the web client is restricted to byond members only."

```

Comment:
It might make players angry if you don't explain why the web client is behind a paywall. Maybe it could be explained here?